include/zephyr/drivers/i3c.h

```diff
@@ -864,11 +864,12 @@ __subsystem struct i3c_driver_api {
 	 * @param dev Pointer to the controller device driver instance.
 	 * @param buf Pointer to the buffer
 	 * @param len Length of the buffer
+	 * @param hdr_mode HDR mode of the buffer
```

Comment:
Should it really say "... of the buffer"?

Reply:
yeah, you're probably right, will remove